sync.go

```diff
@@ -20,6 +20,14 @@ type GHFile struct {
 	URL  string `json:"download_url"`
 }
 
+func getToken() string {
+	if os.Getenv("GITHUB_TOKEN") == "" {
+		return os.Getenv("GITHUB_PACKAGE_PULL_TOKEN")
```

Comment:
May want to preference `GITHUB_PACKAGE_PULL_TOKEN` as people may have old `GITHUB_TOKEN`s laying around as they rotate tokens.

Reply:
Good call, should we just always prefer that? I wish there was a better way to signal this? Maybe an explicit config?

Reply:
Probably always prefer, or just log/error with instructions about the envar name change?

Reply:
The check-access is in local, so each project that relies on this _could_  add that as a prereq for rules that need them like [this](https://github.com/reverbdotcom/reverb/blob/master/Makefile#L280-L281).

Reply:
I just added a write/read to `$HOME/.protopkg` for storing this value.